chatbot_app/startup.py

```diff
@@ -5,4 +5,9 @@
 
 
 def get_connection_string():
-    return os.getenv("CONNECTION_STRING")
\ No newline at end of file
+    return os.getenv("CONNECTION_STRING")
+
+
+SECRET_KEY=os.getenv("SECRET_KEY")
+ALGORITHM=os.getenv("ALGORITHM")
+ACCESS_TOKEN_EXPIRE_MINUTES=int(os.getenv("ACCESS_TOKEN_EXPIRE_MINUTES"))
```

Comment:
dodaj zabezpiecznenie bo jak ktos nie ustawi tych zmiennych to co wtedy? 
Warto rzucic jakims wyjatkiem mysle z sensownym komunikatem